src/main/java/com/CoachPlan/CoachPlanController.java

```diff
@@ -31,13 +31,13 @@ public String index() {
 		return "index";
 	}
 	
-	//this will be the initial athlete page when logged in as an athlete
+	//initial athlete page when logged in as an athlete

```

Comment:
Since this is a method, use proper JavaDoc comments, instead of inline comments.  https://www.baeldung.com/javadoc